src/index.js

```diff
@@ -11,13 +11,13 @@
 /**
  * Blacklist.
  */
-
-const blacklist = ['078051120', '219099999', '457555462'];
+const repitedNumbers = Array.from({ length: 9 - 0 + 1 }, (_, i) => i).map(current => String(current).repeat(9));
+const blacklist = ['078051120', '219099999', '457555462'].concat(repitedNumbers);
+const sequence = Array.from({ length: 9 - 0 + 1 }, (_, i) => i).reduce((acc, current) => acc + current, '').repeat(2);
```

Comment:
```suggestion
const sequence = Array.from({ length: 10 }, (_, i) => i).reduce((acc, current) => acc + current, '').repeat(2);
```

Reply:
Not sure if we should include descending too

Reply:
https://apidocs.taxbit.com/docs/certifying-tax-documentation#taxid-string-required

> The taxId is fully ascending or descending e.g. "123456789" or "987654321"
> The taxId is a single digit repeated e.g. "111111111"